doc/changelog.md

```diff
@@ -34,6 +34,8 @@
   the python function being called back into.
   [(#919)](https://github.com/PennyLaneAI/catalyst/pull/919)
 
+* Autograph now supports static slices. [(#843)](https://github.com/PennyLaneAI/catalyst/pull/843)
```

Comment:
This needs a bit more context, static slices where?

Reply:
Thanks. I have updated this and added an example.